### ISSUE TITLE
Cluster setup problems

### ISSUE THREAD

**Issue by jefflill** [nforgeio/neonKUBE#1258]
We're seeing several cluster setup issues related to `SetupController` and the new support for Hyper-V internal switches.  The `SetupController` issues seem to be related to some changes I made a several weeks back to support cluster setup from within a UI and the switch problems look to be due to using in incorrect Powershell cmdlet namespace as well as the need to better manage the presence of the switch address and NAT.

- [x] The **New-NetNat** command is in the **NetNet** namespace not **NetTCPIP**.

- [x] **Powershell.cs:** Use **-NonInteractive** when executing commands to fail instead of waiting for user interactions.

- [x] **Powershell.cs:** We're seeing TTY commands in the powershell output.  Is there a way of disabling this?

    **JEFF UPDATE:** There doesn't seem to be an obvious way to disable this behavior.  I added the **-NoProfile** option to the powershell command line, but that didn't help.  This isn't super critical right now.  I'm going to create another issue to track this separately: 

    #1259

- [x] **SetupController:** Deployment failures are incorrectly being reported as **cancelled**.

    **JEFF UPDATE:** This was due to having the output strings reversed in the neon-cli cluster prepare and setup commands.

- [x] **SetupController:** It appears that the global image download step is being executed in parallel with the global **PrepareHyperVAsync** step.  This shouldn't be happening.

    **JEFF UPDATE:** This was a problem in `SetupController.ExecuteStep()` which had some code that tried to determine that the step had actually finished execution by observing the node status rather than waiting for the step execution thread to complete.  I changed this so just wait for the thread.

- [x] We need to manage the cluster IP and NAT better:
    - [x] Detect if either of these already exist and/or are in use by something other than our **neonkube** switch.
    - [x] Remove these when uninstalling neonDESKTOP

**Comment by jefflill** [comment]
**FIXED**
